file_unix_test.go

```diff
@@ -21,3 +26,26 @@ func (s *unixFileSuite) TestEnsureBaseDir(c *gc.C) {
 	c.Assert(utils.EnsureBaseDir(`/`, `/b/c`), gc.Equals, `/b/c`)
 	c.Assert(utils.EnsureBaseDir(``, `/b/c`), gc.Equals, `/b/c`)
 }
+
+func (s *unixFileSuite) TestFileOwner(c *gc.C) {
+	username, err := utils.LocalUsername()
+	c.Assert(err, gc.IsNil)
+
+	path := filepath.Join(os.TempDir(), fmt.Sprintf("file-%d", time.Now().Unix()))
+	_, err = os.Create(path)
+	c.Assert(err, gc.IsNil)
+
+	ok, err := utils.IsFileOwner(path, username)
+	c.Assert(err, gc.IsNil)
+	c.Assert(ok, gc.Equals, true)
+}
+
+func (s *unixFileSuite) TestFileOwnerUsingRoot(c *gc.C) {
+	path := filepath.Join(os.TempDir(), fmt.Sprintf("file-%d", time.Now().Unix()))
+	_, err := os.Create(path)
+	c.Assert(err, gc.IsNil)
+
+	ok, err := utils.IsFileOwner(path, "root")
+	c.Assert(err, gc.IsNil)
+	c.Assert(ok, gc.Equals, false)
+}
```

Comment:
Pedantry, but should we test the negative too?

Reply:
Fixed in https://github.com/juju/utils/pull/301/commits/e53e7587a86d6c494e0f5cae7e0d27f22a7487a1